lib/web_ui/lib/src/engine/text_editing/text_editing.dart

```diff
@@ -321,6 +321,14 @@ class EngineAutofillForm {
   }
 
   void placeForm(DomHTMLElement mainTextEditingElement) {
+    // Since we're disabling pointer events on the form to fix Safari autofill,
+    // we need to explicitly set pointer events on the active input element in
+    // order to calculate the correct pointer event offsets.
+    // See: https://github.com/flutter/flutter/issues/136006
+    if(textEditing.strategy is SafariDesktopTextEditingStrategy) {
+      mainTextEditingElement.style.pointerEvents = 'all';
```

Comment:
I have two questions here:

1. Do we need to set this back to `none` after the input loses focus?
2. Should we preemptively set `pointer-events: all` for all inputs in `EngineAutofillForm.fromFrameworkMessage()`?

Reply:
1. We only need to ensure that it remains `none` if it remains on the screen and another input on the form is focused.  This is being [automatically handled for us](https://github.com/flutter/engine/blob/main/lib/web_ui/lib/src/engine/text_editing/text_editing.dart#L258-L260) in `fromFrameworkMessage` inside `EngineAutofillForm`.  This happens when we click any other input in the form.  The other case where it loses focus is where we just click anywhere outside of the input and the form.  In this case, the input [gets moved offscreen](https://github.com/flutter/engine/blob/main/lib/web_ui/lib/src/engine/text_editing/text_editing.dart#L1367) so we don't have to worry about it. 
2. We can't do this because of the Safari autofill fix where we need all the non-active inputs to be on screen but hidden from the user. We actually preemptively disable pointer events for non active inputs in `EngineAutofillForm.fromFrameworkMessage()` so that we don't have pointer collisions if users inadvertently hover/click over those hidden inputs

Reply:
Okay that makes sense. Thanks for explaining this!